Successful Fixes:
- test_batch_size_extremes in tests/test_train.py (Branch: fix-test_batch_size_extremes-84178ff7)
- test_batch_size_extremes in tests/test_train.py (Branch: fix-test_batch_size_extremes-e4bae06a)
- test_learning_rate_extremes in tests/test_train.py (Branch: fix-test_learning_rate_extremes-05364eb4)
- test_learning_rate_extremes in tests/test_train.py (Branch: fix-test_learning_rate_extremes-73488a07)
- test_valid_batch_sizes in tests/test_train.py (Branch: fix-test_valid_batch_sizes-2b282bbf)
- test_valid_learning_rates in tests/test_train.py (Branch: fix-test_valid_learning_rates-0df0c4fd)
- test_tensorboard_logging in tests/test_train.py (Branch: fix-test_tensorboard_logging-cd5cee33)

Failed Fixes:
- test_benchmark_model_model_error in tests/test_benchmark.py (Branch: fix-test_benchmark_model_model_error-6bba1e39)
- test_end_to_end in tests/test_end_to_end.py (Branch: fix-test_end_to_end-a8a71a6c)
- test_logging in tests/test_train.py (Branch: fix-test_logging-d2e6714c)
- test_gpu_not_available in tests/test_train.py (Branch: fix-test_gpu_not_available-847baed2)
- test_end_to_end_training in tests/test_train.py (Branch: fix-test_end_to_end_training-a8a2e4f0)
- assert_called_once_with in /opt/homebrew/Cellar/python@3.12/3.12.5/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/mock.py (Branch: fix-assert_called_once_with-19af4665)


--- Fix Report: 2024-09-20 16:40:35 ---
Successful Fixes:
- test_batch_size_extremes in tests/test_train.py (Branch: fix-test_batch_size_extremes-84178ff7)
- test_batch_size_extremes in tests/test_train.py (Branch: fix-test_batch_size_extremes-e4bae06a)
- test_learning_rate_extremes in tests/test_train.py (Branch: fix-test_learning_rate_extremes-05364eb4)
- test_learning_rate_extremes in tests/test_train.py (Branch: fix-test_learning_rate_extremes-73488a07)
- test_valid_batch_sizes in tests/test_train.py (Branch: fix-test_valid_batch_sizes-2b282bbf)
- test_valid_learning_rates in tests/test_train.py (Branch: fix-test_valid_learning_rates-0df0c4fd)
- test_tensorboard_logging in tests/test_train.py (Branch: fix-test_tensorboard_logging-cd5cee33)

Failed Fixes:
- test_benchmark_model_model_error in tests/test_benchmark.py (Branch: fix-test_benchmark_model_model_error-6bba1e39)
- test_end_to_end in tests/test_end_to_end.py (Branch: fix-test_end_to_end-a8a71a6c)
- test_logging in tests/test_train.py (Branch: fix-test_logging-d2e6714c)
- test_gpu_not_available in tests/test_train.py (Branch: fix-test_gpu_not_available-847baed2)
- test_end_to_end_training in tests/test_train.py (Branch: fix-test_end_to_end_training-a8a2e4f0)
- assert_called_once_with in /opt/homebrew/Cellar/python@3.12/3.12.5/Frameworks/Python.framework/Versions/3.12/lib/python3.12/unittest/mock.py (Branch: fix-assert_called_once_with-19af4665)
